### PR TITLE
Book task 6

### PR DESCRIPTION
**Pull request by MolchanovAlexander**

### CODE REVIEW

**Review by azzilian — COMMENTED** [review]
Gj!

File: src/main/java/com/example/bookstorewebapp/controller/BookController.java [hunk 1]
```diff
@@ -4,51 +4,67 @@
...
+    @Operation(summary = "Get book by id", description = "Get book by id from url /id")
     @GetMapping("/{id}")
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
+    @Operation(summary = "Search book",
+            description = "Search book by author and/or title")
     @GetMapping("/search")
     public List<BookDto> search(BookSearchParameters searchParameters) {
```
Comment (yevheniisynytsia) [inline src/main/java/com/example/bookstorewebapp/controller/BookController.java]: let's add pageable here as well

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Great job! Left one comment

**Review by andrii-hoienko — APPROVED** [review]
Gj!

**Pull request merged**